src/components/Card.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Card = (params) => {
+  const { country, todayCases, countryInfo: { flag } } = params.data; // eslint-disable-line
+  return (
+    <div className="tc bg--light-blue dib pa1 grow bw2 shadow-5 w-48-ns">
```

Comment:
Your design looks good but you are not following the Nelson Sakwa design. Please try adding the corresponding CSS properties (Font size, spacing, etc) to make your application have a similar style to the given one.

Check the image below for a better understanding:
![Capture](https://user-images.githubusercontent.com/75709904/149634854-3da14c51-4fe8-45c2-a7a6-1fffbb709669.PNG)